include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -158,12 +167,26 @@ struct CShuffleEpilogue
         // Store the tile data to the permuted location
         if constexpr(kPadM || kPadN)
```

Comment:
By the way do we really need here this check? The `*_raw` version of tile API just does things using assembly... I'm not sure if we really need it here. The _plain_ tile API should work as well regardless of padding.

Reply:
Are you sure we need it here? Looks like it could improve performance like for example here: https://github.com/ROCm/composable_kernel/pull/1752